app/refinements/string_to_bool.rb

```diff
@@ -0,0 +1,7 @@
+module StringToBool
```

Comment:
should we call this `StringToBoolean` ?

Reply:
I like that it reflect the method that gets added, i.e.`String#to_bool` so if we rename it, we should rename the method too.